lib/index.js

```diff
@@ -1,5 +1,3 @@
-'use strict';
-
 /**
```

Comment:
since this file is now moved, don't we need an index.js file that requires and re-exports or will it be a "top level" export magically? It'd be weird if people had to import `$module/lib`, right?

Reply:
Nevermind, the `main` in the package.json points to the correct file

Reply:
Exactly.